lambda_functions/v1/functions/lpa_codes/app/api/lets_see_about_this.py

```diff
@@ -0,0 +1,54 @@
+import boto3
+
+from . import code_generator
+from .helpers import custom_logger
+import os
+
+logger = custom_logger()
+
+
+def db_connection():
+    if os.environ.get("ENVIRONMENT") in ["ci", "local"]:
+        conn = boto3.resource("dynamodb", endpoint_url="http://localhost:8000")
```

Comment:
Pretty sure this needs region or it will whinge about it.

Reply:
Probably right - will fix in next PR